acceptance/cmd_server_test.go

```diff
@@ -14,11 +14,7 @@ import (
 
 func StartCmdServer(t *testing.T) *testserver.Server {
 	server := testserver.New(t)
-
-	// {$} is a wildcard that only matches the end of the URL. We explicitly use
-	// /{$} to disambiguate it from the generic handler for '/' which is used to
```

Comment:
Don't we need to do this disambiguation anymore?

Reply:
This was specific to built-in Mux, does not work with gorilla anymore.